src/main/java/com/sap/oss/phosphor/fosstars/tool/github/SecurityRatingCalculator.java

```diff
@@ -240,18 +260,45 @@ void run() throws IOException {
     }
   }
 
+  /**
+   * Checks command-line options and throws an exception if something is wrong.
+   *
+   * @param commandLine The command-line options.
+   * @throws IllegalArgumentException If the options are invalid.
+   */
+  private static void checkOptionsIn(CommandLine commandLine) {
+    if (!commandLine.hasOption("url") && !commandLine.hasOption("config")
+        && !commandLine.hasOption("gav")) {
+
+      throw new IllegalArgumentException(
+          "You have to give me either --url, --gav or --config option!");
+    }
+
+    if (commandLine.hasOption("report-type") && !commandLine.hasOption("report-file")) {
```

Comment:
Will approve the PR but I recommend to extract the parameter names/re-used strings like this to constants.

Reply:
I know that such constants may make life easier if we'd like to change its value. But I find it more readable if we have the values here instead of constant names. The parameters of the command-line tool don't change often.